bin/dotd

```diff
@@ -129,7 +135,7 @@ end
 def should_i(question)
   loop do
     print "Should I #{question}? [#{underline 'Y'}es]/#{underline 'S'}kip/#{underline 'Q'}uit: "
-    response = gets.strip.downcase
+    response = flush_and_gets.strip.downcase
     puts ''
     if 'yes'.start_with?(response) # also catches blank/return ;)
```

Comment:
I usually like the option to just hit enter to proceed. It's probably worth changing the logic for the DTP confirmation prompt, but not for every step.